docs/messages/README.md

```diff
@@ -17,33 +17,99 @@ The protocol comprises various messages, which can be classified as either _requ
   - **REFRESH_FLAVOR**, sent by the provider to refresh a particular flavor. By sending a refresh message, the provider helps maintain the availability of flavors and allows the consumer to effectively manage and allocate resources based on the updated expiration time.
   - **WITHDRAW_FLAVOR**, sent by the provider to the consumer to notify that a specific flavor is no longer available. This message serves as a notification mechanism to inform the consumer that the requested flavor is no longer available.
 
+# AUTHENTICATION
+
+The REAR protocol foresees a peer-to-peer interaction between the **consumer** (i.e., the node requesting a given resource) and the **provider** (i.e., the node offering the reaource). 
+Both parties must be authenticated to prevent unauthorized access to the resources available in the continuum.
+
+At it's core, REAR relies on a **Decentralized identifiers (DIDs)** system as a way to provide globally unique identifier to verify the various actors involved in the REAR message exchange and removing the need for a centralized registry.
+Each device in the continuum is represented by a set of attributes, constituting what's called a **Verifiable Credential (VC)**.
+Upon connecting to the continuum, a node must performe the enrolment towards a Trusted Issuer to be included in the DID, receiving as the VC with the attributes disclodesed during the enrolment (see Figure 2).  
```

Comment:
I think that it could be changed by:

...towards a Trusted Issuer in order to obtain a Verifiable Credential, which it can be presented to any other entity in the system, hiding the credential's attributes as deemed necessary for each process. This way, attribute-based access policies can be applied while preserving sensitive information.